src/Grid/grid.jl

```diff
@@ -68,6 +69,16 @@ Note that the vertices are sufficient to define an edge uniquely.
 """
 edges(::AbstractCell)
 
+"""
+    reference_faces(::AbstractRefShape)
+
+Returns a tuple of n-tuples containing the ordered local node indices corresponding to
+the vertices that define an *oriented face*.
```

Comment:
Can you give a reference to oriented face in the manual? Users might not be familiar with the concept.